### ISSUE TITLE
📑 Feature Request: Incorporate a new function to utils of Ersilia to get the Python Path of a Conda environment.

### ISSUE THREAD

**Issue by carcablop** [ersilia-os/ersilia#521]
### Is your feature request related to a problem? Please describe.

When I tried to test a model, this was generating an error and it was not running inside the conda environment that was created from the model when I did the fetch, but it was running in another conda environment, this generates problems with dependencies, since these dependencies were only installed in the environment created when you fetched. So it was necessary to specify the python path of the conda environment of the model and pass it to the function so that it could be executed without errors.

### Describe the solution you'd like.

In the utils folder of the Ersilia repository, in the script conda.py, we should have a function that returns the python path given a conda environment.

A function like this: 
```

`def getPythonPath_env(name_env_model):
    python_path= subprocess.check_output("which python", shell=True).strip()
    index_env=python_path.decode('utf-8').find("envs/") + 5
    python_path_envs_model= python_path.decode('utf-8')[0:index_env]  
    return python_path_envs_model + name_env_model + "/bin/python"`
```

**Comment by carcablop** [comment]
Hello. 
This is the pull request to incorporate the function. 
https://github.com/ersilia-os/ersilia/pull/522

Thank you.

**Comment by carcablop** [comment]
> 
Hi Gemma, the intention of adding the function is to incorporate it as a utility for future implementations, in cases where it is required to interact with conda through python. 
I made some corrections and made some changes suggested by Miquel. This is the new pull request[ ]( https://github.com/ersilia-os/ersilia/pull/525)

**Comment by GemmaTuron** [comment]
I've merged PR #525 ! thanks @carcablop
